app/src/main/kotlin/lv/rigadevday/android/ui/venues/VenueFragment.kt

```diff
@@ -0,0 +1,64 @@
+package lv.rigadevday.android.ui.venues
+
+import android.os.Bundle
+import android.text.Html
+import android.view.View
+
+import kotlinx.android.synthetic.main.fragment_venue.view.*
+import lv.rigadevday.android.R
+import lv.rigadevday.android.ui.base.BaseFragment
+import lv.rigadevday.android.ui.openEmail
+import lv.rigadevday.android.ui.openMap
+import lv.rigadevday.android.ui.openWeb
+import lv.rigadevday.android.utils.BaseApp
+import lv.rigadevday.android.utils.toExtraKey
+
+class VenueFragment : BaseFragment() {
+
+    val EXTRA_VENUE_INDEX = "venue_index".toExtraKey()
+
+    companion object {
+        fun newInstance(index: Int) = VenueFragment().apply {
+            arguments = Bundle().apply {
+                putInt(EXTRA_VENUE_INDEX, index)
+            }
+        }
+    }
+
+    override val layoutId = R.layout.fragment_venue
+
+    override fun inject() {
+        BaseApp.graph.inject(this)
+    }
+
+    override fun viewReady(view: View) {
+
+        val index = arguments.getInt(EXTRA_VENUE_INDEX)
+
+        repo.venue(index).subscribe(
+                { venue ->
+                    view.venue_name.text = venue.name
```

Comment:
You can create function 
```
fun poplateView(view: View) = with(view) {
    /*code here*/
}
``` 
to reduce indentation. As a bonus you will not need to write `view` all the time.